iOS document (#76)
mirrorworld-universe/docs.mirrorworld.fun: data/ios/ios-authentication.mdx

# Authentication

> Token saved in local by SDK would be out of date. So when a user opens the app, the developer should check his authentication every time. If token is invalidate, we should guild users to login again.

Reviewer: ```suggestion
Token saved locally by SDK would be out of date. So when a user opens the app, they should check their authentication every time. If the token is invalidated, we should guide users to login again.
```
